css/src/atomics/border.scss

```diff
@@ -0,0 +1,59 @@
+$border-positions: (('top', 'right', 'bottom', 'left'));
```

Comment:
What's with the double `((`?

Reply:
Hmm, good catch! Honestly, I borrowed that approach from `spacing.scss` file and didn't notice the double bracket. I'll fix it in both files

Reply:
Fixed